tasty-plutus/src/Test/Tasty/Plutus/Internal/Context.hs

```diff
@@ -208,6 +215,41 @@ compileMinting conf cb =
     go :: TxInfo
     go = baseTxInfo conf cb
 
+{- | Combine a list of partial contexts that should,
+     when combined, validate, but fail when any one
+     partial context is missing. The input is a list
+     of pairs where the first element is the partial
+     context, and the second is the test message when
+     that particular context is missing. e.g.
+
+     >>> makeIncompleteContexts
```

Comment:
Does this look OK in the resulting Haddocks? I'd normally use 'Bird tracks', aka

```
> module Foo where
> 
> import Bar (Baz)
> 
> ...
```

Reply:
It looked fine, but I changed to bird tracks anyways and made it slightly clearer what the code meant.